pkg/version/version.go

```diff
@@ -45,6 +45,7 @@ func CheckClientVersion(expectedVersion string) error {
 	return nil
 }
 
+//
```

Comment:
Looks like this refers to the version I had before I force-pushed adding @siggy suggested changes and removing mine. Am I breaking etiquette by doing force-pushes into PRs?

Reply:
@alpeb No worries! In general we try to avoid force-pushes in PR, to show reviewers the incremental changes as we go along.

That said, I think this case is an exception where the initial PR was branched off an older master, already had conflicts, and the subsequent commit was quite different from the initial one. I probably would have force-pushed in this case.